save-frontend/src/main/kotlin/org/cqfn/save/frontend/components/basic/ExecutionStatistics.kt

```diff
@@ -71,5 +71,11 @@ fun executionTestsNotFound(count: Int?) = fc<ExecutionStatisticsProps> {
                 +"Tests not found!"
             }
         }
+    } else if (count == 0 && props.executionDto?.status == ExecutionStatus.PENDING) {
+        div("d-sm-flex align-items-center justify-content-center mb-4 mt-2") {
+            h1("h3 mb-0 text-gray-800") {
+                +"Loading ..."
```

Comment:
```suggestion
                +"Execution is starting..."
```
It's not exactly 'loading', because while execution is still pending, there is nothing to load :)

Reply:
Agree, fixed)